l10n/en_GB.js

```diff
@@ -74,6 +74,6 @@ OC.L10N.register(
     "Shared boards" : "Shared boards",
     "View more" : "View more",
     "Move board to archive" : "Move board to archive",
-    "Create a new board" : "Create a new board"
+	"Create a new board" : "Create a new board"	
```

Comment:
This change can be removed as well.

Reply:
Isn't this string part of the l10n definitions? I see it in every other file, all I did here was remove the comma I appended at the end of the line, and obvious I accidentally added in some extra white space.

Reply:
I was referring to the whitespace :wink: But that's a non-blocker.